### DISCUSSION TITLE
Should we release 0.72.12?

### DISCUSSION THREAD

**Discussion by huntie** [reactwg/react-native-releases#108] [Patches]
# All pick requests should now be filed through the [issue template](https://github.com/reactwg/react-native-releases/issues/new?assignees=&labels=Type%3A+Pick+Request&projects=&template=pick_request_form.yml)

You can see outstanding pick requests for 0.72.x in its [dedicated project board](https://github.com/orgs/reactwg/projects/7)

See https://github.com/reactwg/react-native-releases/discussions/117 for more details

We will lock this discussion for now.

---

## Should we release 0.72.12?

Current Release: [0.72.11](https://github.com/facebook/react-native/releases/tag/v0.72.11)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

**Comment by cipolleschi** [comment]
Please, merge https://github.com/facebook/react-native/pull/43247
This solve a possible ENV var clash between a RN env var and other env vars users can set in their system.

**Reply by huntie** [reply]
Merged ✅

**Comment by cipolleschi** [comment]
Can we merge https://github.com/facebook/react-native/pull/43240?
This solve a problem we were facing with libraries using `install_modules_dependencies` and it has already been validated with 0.73

**Reply by huntie** [reply]
Merged ✅

**Comment by GijsWeterings** [comment]
(Copying from the top post, in case people scroll down immediately):

All pick requests should now be filed through the [issue template](https://github.com/reactwg/react-native-releases/issues/new?assignees=&labels=Type%3A+Pick+Request&projects=&template=pick_request_form.yml)

You can see outstanding pick requests for 0.72.x in its [dedicated project board](https://github.com/orgs/reactwg/projects/7)

See https://github.com/reactwg/react-native-releases/discussions/117 for more details

We will lock this discussion for now.
